code/game/objects/items.dm

```diff
@@ -53,6 +52,10 @@
 	var/vending_cat = null// subcategory for vending machines.
 	var/list/dynamic_overlay[0] //For items which need to slightly alter their on-mob appearance while being worn.
 
+	var/is_shrapnel = 0
```

Comment:
Aight. I think it would be a good idea to add a shrapnel size aswell, so that glass shards could take up 2 slots and produce 3-5 shrapnel while an empty bullet casing or something could take 1 slot and produce 1 shrapnel.